.github/workflows/scripts/kinds/verify-kinds.go

```diff
@@ -414,3 +311,113 @@ func contains(array []string, value string) bool {
 
 	return false
 }
+
+type kindRegistry struct {
+	zipDir string
+}
+
+func NewKindRegistry() (*kindRegistry, error) {
+	ctx := context.Background()
+	tc := oauth2.NewClient(ctx, nil)
+	client := github.NewClient(tc)
+
+	// Create a temporary file to store the downloaded archive
+	file, err := ioutil.TempFile("", "*.zip")
```

Comment:
This function is deprecated in favor of [`os.CreateTemp()`](https://pkg.go.dev/os#CreateTemp) :)